components/raftstore/src/store/fsm/peer.rs

```diff
@@ -876,6 +877,7 @@ where
             return;
         }
         let target_index = if self.fsm.peer.force_leader.is_some() {
+            FORCE_LEADER_CHECK.inc();
```

Comment:
Maybe this metric should be more generic to record all kinds of Unsafe Recovery events. You can refer to https://github.com/tikv/tikv/blob/6c06f99c1b9dc10811e689cfc979a1b8d4287dd6/components/raftstore/src/store/metrics.rs#L687 to create something like `UnsafeRecoveryEventCounterVec`? cc @Connor1996

Reply:
use `self.ctx.raft_metrics.invalid_proposal` is enough